src/ophyd_epics_devices/panda.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
why are we importing annotations here if it isn't being used?

Reply:
Yeah, looks like this isn't actually required right now

Reply:
This import makes `__annotations__` be stored in string representation: https://docs.python.org/3/library/__future__.html
It is generally useful so that all annotations are lazily evaluated. It will also become the default in a later version of Python, so is good practice to put in for anything dealing with type hints.

Reply:
After doing some digging:
When we use `get_type_hints` with an object (rather than type(object)), the function will use the object's name space when searching for type hints. If the type hints were evaluated immediately (i.e this import isn't used), this is fine. If the import IS used, the type hints are still strings by the time this function is being run, and the object namespace doesn't contain the right definitions. (Using type(obj) finds a different namespace which is why that fixed the error.

Ensuring we use the argument `globalns=globals()` every time we do `get_type_hints` will fix this, and seems neater than adding the extra `get_type_hints_with_inheritance `function